src/bitmessageqt/dialogs.py

```diff
@@ -2,7 +2,7 @@
 src/bitmessageqt/dialogs.py
 ===========================
 """
-
+# pylint: disable=relative-import,too-few-public-methods
```

Comment:
`relative-import` isn't needed. Please make sure your code quality tools use the configuration from the project root.